lib/css/components/tables.less

```diff
@@ -1,313 +1,296 @@
-//  ============================================================================
-//  $    CONTAINER
-//  Give tables an optional container for responsive scrolling
-//  ----------------------------------------------------------------------------
 .s-table-container {
     overflow-x: auto;
     @scrollbar-styles();
 }
 
-//  ============================================================================
-//  $    BASE TABLE STYLE
-//  ----------------------------------------------------------------------------
 .s-table {
-    display: table;
-    width: 100%;
-    max-width: 100%;
-    border-collapse: collapse;
-    border-spacing: 0;
-    font-size: var(--fs-body1);
-
-    th,
-    td {
-        padding: var(--su8);
-        border-top: 1px solid var(--bc-medium);
-        border-left: 1px solid var(--bc-medium);
-        border-right: 1px solid var(--bc-medium);
-        vertical-align: middle;
-        color: var(--fc-medium);
-        text-align: left;
-
-        &.s-table--bulk {
-            width: calc(var(--su32) - var(--su2));
-        }
+    @ta-cell-border: var(--su-static1) solid var(--bc-medium);
+    @ta-columns: (100% / 12);
+
+    --_ta-tbody-tbody-bc: var(--bc-medium);
+    --_ta-tbody-tbody-bw: var(--su-static2);
+    --_ta-tbody-tr-even-bg: unset;
+    --_ta-td-bbw: 0;
+    --_ta-td-bc: var(--bc-medium);
+    --_ta-td-fs: unset;
+    --_ta-td-fw: unset;
+    --_ta-td-p: var(--su8);
+    --_ta-td-ta: left;
+    --_ta-td-va: middle;
+    --_ta-th-w: unset;
+    --_ta-thead-th-bg: var(--black-025);
+    --_ta-th-bbw: 0;
+    --_ta-th-bc: var(--bc-medium);
+    --_ta-th-fs: unset;
+    --_ta-th-p: var(--su8);
+    --_ta-th-ta: left;
+    --_ta-th-va: middle;
+    --_ta-th-w: unset;
```

Comment:
Looks like a duplicate of line 20

Reply:
Oh, I believe the custom property on line 20 should be `--_ta-td-w`. Good catch!